pyntcloud/__init__.py

```diff
@@ -1,7 +1,11 @@
-from .core_class import PyntCloud
-
 MAJOR = 0
 MINOR = 1
 MICRO = 0
 
 __version__ = "{}.{}.{}".format(MAJOR, MINOR, MICRO)
+
+
+try:
```

Comment:
Have you added this for debugging and forgot to revert ii or this try/except catch is here for a reason??

Reply:
The try/except ImportError fixes the issue during setup. I've seen in used in other libraries.

I would have like to do something like this:

```
try:
    from .core_class import PyntCloud
except ImportError:
    raise RuntimeError('Error: numpy not installed. Use pip install numpy')
```

However, during setup that RuntimeError is never raised. Setup automatically installs numpy and everything proceeds normally.

So the question in my mind was- Is it better to raise a dummy runtime error or simply use pass?

I chose pass because, I'm not able to determine which dependency is failing. Maybe a small comment alongside pass would help.

Let me know what you'd prefer to do. I'm happy to make the necessary change.

Regards,
Pandu

Reply:
I understand. However there is a better solution to this problem: removing the import in setup.py https://github.com/daavoo/pyntcloud/blob/master/setup.py#L1

That import is only being used for the versioning (I actually feel ashamed for not realizing this xD) so you can replace it hardcoding the version string in setup.py (I don't care about versioning right now, although I'm planning to care in the near future) and fix the problem which is causing setup.py to fail because it tries to import `PyntCloud` unleashing a cascade of imports which fail if the dependencies are not installed.

If you do that, you will remove this bad behaviour regarding the cascade of imports and you could remove the try/except block because it won't be reached anymore during setup.